LICENSE.md

```diff
@@ -6,7 +6,7 @@ Emperor makes usage of all of the following copyrighted packages; their licenses
 --------------------------------------------------------------------------------
 
 #### SlickGrid (v2.1.0-dev)
-[SlickGrid](https://github.com/mleibman/SlickGrid) - A lightning fast JavaScript grid/spreadsheet (e004912b5ce29ac0d0cb04df50fe66db5e3af9ea)
+[SlickGrid](https://github.com/6pac/SlickGrid) - A lightning fast JavaScript grid/spreadsheet (e004912b5ce29ac0d0cb04df50fe66db5e3af9ea)
```

Comment:
We should probably update the SHA to point to d05dcd5b9369cae4b61df895ef7f51e8248e11c2 i.e. the SHA on @6pac's fork.